core/src/main/java/net/sourceforge/jnlp/runtime/classloader/DelegatingClassLoader.java

```diff
@@ -0,0 +1,73 @@
+package net.sourceforge.jnlp.runtime.classloader;
+
+import java.net.URL;
+import java.util.Objects;
+
+public class DelegatingClassLoader extends ClassLoader
+{
```

Comment:
your format does not belong the the format that we use in the project . All '{' should be at the end of lines, for example.

Reply:
I will change it manually, i would recommend to add a .editorconfig file to the repo so this problem will not occur anymore. https://editorconfig.org/

Reply:
fixed

Reply:
https://editorconfig.org/ looks like a good improvement. @sclassen what do you think? @Gr33nbl00d are you familiar with that and could provide a PR for it?

Reply:
Personally i use Intellij IDEA and there is an option to export the code style directly to .editorconfig. I guess that such an option is available for other IDEs too. So in best case you simply need to export your current config to .editorconfig and commit it to the repo.

Reply:
We use IntelliJ too. Can you give us link to a manual how to do that?

Reply:
![image](https://user-images.githubusercontent.com/22657661/89023461-e6570480-d323-11ea-914c-eadf69ef277a.png)

In the settings dialog a bit hidden maybe :)

Reply:
https://github.com/karakun/OpenWebStart/issues/302